src/test/regress/expected/multi_insert_select.out

```diff
@@ -2719,6 +2738,92 @@ SELECT * FROM dist_table_with_sequence ORDER BY user_id, value_1;
 (6 rows)
 
 DROP TABLE dist_table_with_sequence;
+-- Select into distributed table with a user-defined sequence
+CREATE SEQUENCE seq1;
+CREATE SEQUENCE seq2;
+CREATE TABLE dist_table_with_user_sequence (user_id int default nextval('seq1'), value_1 bigint default nextval('seq2'));
+SELECT create_distributed_table('dist_table_with_user_sequence', 'user_id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- from local query
+INSERT INTO dist_table_with_user_sequence (value_1)
+SELECT s FROM generate_series(1,5) s;
+SELECT * FROM dist_table_with_user_sequence ORDER BY user_id, value_1;
+ user_id | value_1
+---------------------------------------------------------------------
+       1 |       1
+       2 |       2
+       3 |       3
+       4 |       4
+       5 |       5
+(5 rows)
+
+-- from a distributed query
+INSERT INTO dist_table_with_user_sequence (value_1)
+SELECT value_1 FROM dist_table_with_user_sequence ORDER BY value_1;
+SELECT * FROM dist_table_with_user_sequence ORDER BY user_id, value_1;
+ user_id | value_1
+---------------------------------------------------------------------
+       1 |       1
+       2 |       2
+       3 |       3
+       4 |       4
+       5 |       5
+       6 |       1
+       7 |       2
+       8 |       3
+       9 |       4
+      10 |       5
+(10 rows)
+
+TRUNCATE dist_table_with_user_sequence;
+INSERT INTO dist_table_with_user_sequence (user_id)
+SELECT user_id FROM raw_events_second ORDER BY user_id;
+SELECT * FROM dist_table_with_user_sequence ORDER BY user_id, value_1;
+ user_id | value_1
+---------------------------------------------------------------------
+       1 |       1
+       2 |       2
+       3 |       3
+       4 |       4
+       5 |       5
+(5 rows)
+
+WITH top10 AS (
+  SELECT user_id FROM raw_events_second WHERE value_1 IS NOT NULL ORDER BY value_1 LIMIT 10
+)
+INSERT INTO dist_table_with_user_sequence (value_1)
+SELECT * FROM top10;
+ERROR:  cannot handle complex subqueries when the router executor is disabled
```

Comment:
this error message is confusing (router executor is enabled here, right?)

probably worth opening an issue

Reply:
> (router executor is enabled here, right?)
Yes it seems to be enabled. I didn't notice this problem since it was also occurring for serial pseudo-types. Will open an issue about it.